potodo/_github.py

```diff
@@ -1,32 +1,45 @@
 import requests
 
-from subprocess import getoutput
+from subprocess import check_output
 
 
-def get_repo_url():
-    url = getoutput("git remote get-url --all upstream")
+def get_repo_url() -> str:
+    """
+    Tries to get the repository url from git commands
+    """
+    url: str = str(check_output("git remote get-url --all upstream"))
     if "fatal" in url:
-        url = getoutput("git remote get-url --all origin")
+        url = str(check_output("git remote get-url --all origin"))
     if "fatal" in url:
+        # If the commands didn't work
         raise ValueError(
             f"Unknown error. `git get-url --all upstream|origin` returned {url}"
         )
     return url
 
 
-def get_repo_name():
+def get_repo_name() -> str:
+    """
+    Will get the repository url from git commands then remove useless stuff to get ORG/NAME
```

Comment:
[Docstring conventions](https://www.python.org/dev/peps/pep-0257/) ask for docstrings to start on the first line.

Reply:
I know but I hate it 😭

Reply:
Try them a month or two, you hate them at the moment, but I bet you can get used to it.

On the long run I think it's better to get used to it: you'll read other people code without hating their code.

Reply:
Alright